diff --git a/node_modules/capacitor-plugin-send-intent/dist/esm/web.js b/node_modules/capacitor-plugin-send-intent/dist/esm/web.js
index 10bf3e5..c1431e3 100644
--- a/node_modules/capacitor-plugin-send-intent/dist/esm/web.js
+++ b/node_modules/capacitor-plugin-send-intent/dist/esm/web.js
@@ -1,14 +1,8 @@
-import { WebPlugin } from '@capacitor/core';
+﻿import { WebPlugin } from '@capacitor/core';
 export class SendIntentWeb extends WebPlugin {
     constructor() {
-        super({
-            name: 'SendIntent',
-            platforms: ['web']
-        });
+        super();
     }
 }
 const SendIntent = new SendIntentWeb();
 export { SendIntent };
-import { registerWebPlugin } from '@capacitor/core';
-registerWebPlugin(SendIntent);
-//# sourceMappingURL=web.js.map
\ No newline at end of file
